.github/workflows/regression-tests.yml

```diff
@@ -42,11 +42,7 @@ jobs:
           dfly-executable: dragonfly
           gspace-secret: ${{ secrets.GSPACES_BOT_DF_BUILD }}
           build-folder-name: build
-          # This expression serves as a ternary operator, i.e. if the condition holds it returns
```

Comment:
what have been fixed here?

Reply:
I added dbg_only instead of NOT_EXISTING_MARKER

Reply:
Dbg_only because the release build is too fast